lib/rucio/core/rse.py

```diff
@@ -161,6 +158,9 @@ def bulk_load(rse_datas: "Sequence[RseData]", load_name=False, load_columns=Fals
                 rse_data.limits = limits
 
 
+RSE_SETTINGS = ["continent", "city", "region_code", "country_name", "time_zone"]
+
+
 @transactional_session
 def add_rse(rse, vo='def', deterministic=True, volatile=False, city=None, region_code=None, country_name=None, continent=None, time_zone=None,
```

Comment:
This will involve a bigger change (Rest/Client Api with deprecation, other places where `add_rse(` is called), I'd like to do that in a follow-up PR once the data is migrated.